.github/workflows/link-checker.yml

```diff
@@ -4,6 +4,9 @@ on:
   # Check links once a week on Monday
   schedule:
     - cron: "0 0 * * 1"
+  pull_request:
+    branches:
+      - "master"
```

Comment:
You now have a commit where this is removed, and another commit where you add it back. I think a bunch of these commits need to be reworked and squashed, or have changes moved between them.

Reply:
You can probably get away with one commit for the links and reference changes, and another commit with just the build script and workflow changes (if that makes things easier).

Reply:
Forget about this, check the commits. Tell me in a new review how I did. I have checked all commits I think they should all make sense. 

TIL `git reset @~`